src/helpers/balloonHelper.ts

```diff
@@ -36,7 +36,11 @@ export function getBalloonPosition(
   if (map instanceof CesiumMap) {
     const wgs84Position = Projection.mercatorToWgs84(position)
     cartesian = Cartographic.toCartesian(
-      Cartographic.fromDegrees(wgs84Position[0], wgs84Position[1])
+      Cartographic.fromDegrees(
+        wgs84Position[0],
+        wgs84Position[1],
+        position[2] ?? 0
```

Comment:
Can you put the default height to 50 please 
position[2] ?? 50
This fixes the bug on this ticket : https://jira.camptocamp.com/browse/GSREN3D-315